Foundation/NSAffineTransform.swift

```diff
@@ -9,7 +9,7 @@
 
 #if os(OSX) || os(iOS)
     import Darwin
-#elseif os(Linux)
+#elseif os(Linux) || os(Android)
```

Comment:
This boilerplate goes away for most users as if they import Foundation as there is an @_exported import of the right C library in NSSwiftRuntime.swift.